local.yml

```diff
@@ -27,8 +26,8 @@ services:
       - ./.envs/.local/.postgres
       - ./.envs/.local/.secrets
     ports:
-      - "8000:8000"
-      - "8001:8001"
+      - "0.0.0.0:8000:8000"
```

Comment:
Wondering why this is needed here? For opening up local dev setup to traffic, I'd prefer to use ngrok or forward a tcp port via ssh. Let me know what issues you were seeing.

Reply:
ngrok or ssh port forwarding is not needed, only local access is needed so that octoprint can reach the dev machine. Why open access over the internet? :thinking:

Reply:
I expose my local dev setup via ngrok to test webhooks end-to-end. I'm cool with making this optional, just wanted to clarify that this is used!

Reply:
oooh, didn't think about it at all. Let me revisit my approach and come up with a better solution